exercises/forth/canonical-data.json

```diff
@@ -0,0 +1,334 @@
+{
+  "#": [
+    "The cases are split into multiple sections, all with the same structure.",
+    "In all cases, the `expected` key is the resulting stack",
+    "after executing the Forth program contained in the `input` key."
+  ],
+  "parsing and numbers": {
+    "cases": [
+      {
+        "description": "empty input results in empty stack",
+        "input": [],
+        "expected": []
+      },
+      {
+        "description": "numbers just get pushed onto the stack",
+        "input": [
+          "1 2 3 4 5"
+        ],
+        "expected": [1, 2, 3, 4, 5]
+      },
+      {
+        "description": "all non-word characters are separators",
+        "input": [
+          "1\u00002\u00133\n4\r5 6\t7"
```

Comment:
Garbage. This \u0013 is a "device control 3". The original https://github.com/exercism/exercism.io/pull/1188 meant for this to be \u0001, "start of heading" (whatever that is).

My fault. Sorry.